nservicebus/messaging/publish-subscribe/publish-handle-event.md

```diff
@@ -11,75 +12,61 @@ related:
 ---
 
 
-## Classifying a message as an event
+## Defining events
 
-To publish a message it must be classified as an event. There are two ways of achieving this
+The messages needs to be declared as an event before they can be published. That can be done with marker interfaces or using message conventions.
 
 
 ### Via a Marker interface
 
-Adding a marker interface to the message definition.
+Add `IEvent` marker interface to the message definition:
 
 snippet:EventWithInterface
 
 
 ### Via Message Conventions
 
-Using the `EventWithConvention` message convention.
-
-Given a message with the following definition.
+The following message:
 
 snippet:EventWithConvention
 
-It could be treated as an event using the following convention.
+can be declared as an event using the following convention:
 
 snippet:DefiningEventsAs
 
+Learn more about [conventions](/nservicebus/messaging/conventions.md) and [unobtrusive mode](/nservicebus/messaging/unobtrusive-mode.md).
 
-## Handling a event
 
-An event can be handled by use of the `IHandleMessages` interface on any [Handler](/nservicebus/handlers) or [Saga](/nservicebus/sagas).
+## Handling an event
 
-## Publishing an event
+In order to handle an event, implement `IHandleMessages<T>` interface in any [Handler](/nservicebus/handlers) or [Saga](/nservicebus/sagas) class, where `T` is the specific event type.
```

Comment:
>In order to handle an event, implement `IHandleMessages<T>` interface in any [Handler]

might be better as:

In order to handle an event, implement `IHandleMessages<T>` interface in any class,  where `T` is the specific event type.  The class can either be a [message handler](/nservicebus/handlers) or a [Saga](/nservicebus/sagas).